tool_collections/cufflinks/cuff_macros.xml

```diff
@@ -1,5 +1,5 @@
 <macros>
-  <token name="@VERSION@">2.2.1</token>
+  <token name="@VERSION@">2.2.1.2</token>
```

Comment:
I think I prefer the previous approach were this is equal to the Cufflinks version and each tool updates its Galaxy version only when needed.